keycastr/ModsVisualizer.h

```diff
@@ -0,0 +1,61 @@
+//	Copyright (c) 2009 Stephen Deken
```

Comment:
Please update source headers to reflect the last year you edited the file, and *your* name, since it's your code ;)